docs/docs/configuration.md

```diff
@@ -308,4 +310,84 @@ modules:
 ```
 When the `no-source` parameter is `true`, the tool does not validate the `path` property, so it can be omitted as shown in the example above. No action is performed during the module's build step, so all parameters configuring the module build behaviour, e.g. `builder`, `timeout`, `ignore`, etc. are ignored. There is no content associated with this module in the result MTA archive. The rest of the module processing is the same as when the parameter is not provided, e.g. the entry in the generated the deployment descriptor is created based on `supported-platforms` settings. 
 
+#### Configuring a module that need to generate SBOM content
+Three native builders `npm, maven and golang` are upgraded to support SBOM generation.
+
+Set `build-parameters.builder` attribute value to `npm, maven or golang` :
+
+```yaml
+modules:
+  - name: node-module
+  type: nodejs
+  path: nodejs
+  provides:
+  - name: node-js_api
+    properties:
+        url: ${default-url}
+  build-parameters:
+    builder: npm
+    requires:
+      - name: java-module
+ 
+- name: java-module
+  type: java
+  path: java
+  properties:
+    MEMORY_CALCULATOR_V1: true
+  build-parameters:
+    builder: maven
+    requires:
+      - name: go-module
+
+- name: go-module
+  type: go
+  path: golang
+  parameters:
+    memory: 512M
+    disk-quota: 256M
+  properties:
+    MEMORY_CALCULATOR_V1: true
+  build-parameters:
+    builder: golang
+    requires:
+      - name: custom-module
+
+mbt build --sbom-file-path sbom-path/test.sbom.xml
+
+or
+
+mbt sbom-gen --sbom-file-path sbom-path/test.sbom.xml
+```
+
```

Comment:
For the `java` or `nodejs` module types, since their default builder is `maven` or `npm`, you don't need to set the `build-parameters.builder` attribute:

Reply:
OK